src/cluster_legacy.c

```diff
@@ -4683,8 +4683,8 @@ void clusterHandleReplicaFailover(void) {
     if (server.cluster->failover_auth_sent == 0) {
         server.cluster->currentEpoch++;
         server.cluster->failover_auth_epoch = server.cluster->currentEpoch;
-        serverLog(LL_NOTICE, "Starting a failover election for epoch %llu.",
-                  (unsigned long long)server.cluster->currentEpoch);
+        serverLog(LL_NOTICE, "Starting a failover election for epoch %llu, node epoch is %llu",
+                  (unsigned long long)server.cluster->currentEpoch, (unsigned long long)nodeEpoch(myself));
```

Comment:
I am struggling with the changes here. If it is not needed, you can use the review suggestion to remove it.
```suggestion
        serverLog(LL_NOTICE, "Starting a failover election for epoch %llu.",
                  (unsigned long long)server.cluster->currentEpoch);
```

Reply:
it doesn't hurt to include node's config epoch but maybe add `config` too? 
```suggestion
        serverLog(LL_NOTICE, "Starting a failover election for epoch %llu, node config epoch is %llu",
                  (unsigned long long)server.cluster->currentEpoch, (unsigned long long)nodeEpoch(myself));
```